docs/user/docs/installation.md

```diff
@@ -0,0 +1,252 @@
+        chmod 600 $HOME/.kube/config # Nobody else should be able to read the configuration
+        ```
+
+=== "k3d"
+
+    We are constantly working on expanding our documentation. This installation method is currently not documented. If it is relevant, please feel free to contact us at set@deutschebahn.com.
+
+=== "OpenShift"
+
+    We are constantly working on expanding our documentation. This installation method is currently not documented. If it is relevant, please feel free to contact us at set@deutschebahn.com.
+
+## Step 2: Validate the available resources
+
+The minimum required resources are 3
+[Kubernetes CPU cores](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-cpu)
+and around 2,5GiB of memory for the management platform. Depending on the load,
+the instance can scale up and is limited to 10 Kubernetes CPU cores cores and
+~8GiB of memory.
+
+Each session requires a minimum of 0.4 Kubernetes CPU cores and 1.6Gi of
+memory. A session can scale up until it reaches 2 Kubernetes CPU cores and 6Gi
+of memory.
+
+## Step 3: Set up the required namespaces (optional)
+
+The Collaboration Manager requires two different namespaces. For security and
+overview reasons, they are separated:
+
+<!-- prettier-ignore -->
+- Capella Collaboration Manager control namespace: In this namespace, we run
+  the core application. It has full control over the sessions namespace and  consists of the following services:
+    - Frontend
+    - Backend
+    - Documentation
+    - Guacamole
+    - Prometheus
+    - Grafana (Loki), can be disabled in the `values.yaml`
+
+- Sessions namespace. The namespace is controlled by the control namespace and you won't need to touch it. In the session namespace, the following services run:
+    - Storage for persistent workspaces
+    - Storage for Juypter file-shares
+    - Pipeline jobs for nightly TeamForCapella to Git synchronisation
+    - Session containers (Capella, Papyrus, Juypter, pure::variants)
+
+<!-- prettier-ignore -->
+1. Create the two required namespaces:
+
+    ```zsh
+    kubectl create namespace collab-manager # If you use another name, please update the following commands and use your namespace name.
+    kubectl create namespace collab-sessions # If you use another name, please update the `values.yaml` accordingly.
+    ```
+
+2. Set the `collab-manager` as default namespace in the default context (optional):
+
+    ```zsh
+    kubectl config set-context --current --namespace=collab-manager
+    ```
+
+## Step 4: Install helm
+
+Follow the official instructions to install Helm:
+[Installing helm](https://helm.sh/docs/intro/install/)
+
+Verify that `helm` is working by executing the command:
+
+```zsh
+helm version
+```
+
+## Step 5: Clone the Github repository
+
+Navigate to a persistent location on your server, e.g. `/opt`. Then clone the
+Github repository by running:
+
+```zsh
+git clone https://github.com/DSD-DBS/capella-collab-manager.git
```

Comment:
what about the docker-images submodule? that is not required because you use the docker images differently?